gns3/items/node_item.py

```diff
@@ -377,6 +377,10 @@ def itemChange(self, change, value):
         :param value: value of the change
         """
 
+        if change == 9 and self._main_window.uiSnapToGridAction.isChecked(): # Unsure but change seems to alternate between 0/9 while dragging
```

Comment:
This documentation will help you to know the name of the event:
http://doc.qt.io/qt-5/qgraphicsitem.html

In your case it's QtWidgets.QGraphicsItem.ItemPositionHasChanged